### PR TITLE
V1.3.1 Gitlab Integration + bug fixes

### PR DESCRIPTION
**Pull request by Feroz-Shaikh**

### CODE REVIEW

File: installscripts/cookbooks/jenkins/files/node/jenkins-conf.properties [hunk 1]
```diff
@@ -0,0 +1,46 @@
+REPO_BASE=
```
Comment (suryajak) [inline installscripts/cookbooks/jenkins/files/node/jenkins-conf.properties]: Why is this file back?

File: installscripts/jazz-terraform-unix-noinstances/scripts/gitlab-trigger-job.sh [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+          <targetBranchRegex></targetBranchRegex>
+          <secretToken>replace</secretToken>
+        </com.dabsquared.gitlabjenkins.GitLabPushTrigger>
+      </triggers>
+    </org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
+  </properties>
+  <definition class="org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition" plugin="workflow-cps@2.36">
+   <script>#!groovy.
+
+node  {
```
Comment (suryajak) [inline installscripts/jazz-terraform-unix-noinstances/scripts/gitlab-trigger-job.sh]: Do we know why this code is in installer? Is this required? if yes, we probably should create a Jenkins file in core and reference it.

Comment (suryajak) [reply installscripts/jazz-terraform-unix-noinstances/scripts/gitlab-trigger-job.sh]: This is a comment for code from 44 - 154

File: installscripts/jazz-terraform-unix-noinstances/scripts/gitlab-trigger-job.sh [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+          //Read the JenkinsFile to find the service name and domain
+           def jenkinsfile_dir = sh (
+                      script: &quot;pwd&quot; ,
+                      returnStdout: true
+              ).trim()
+          myFileName = jenkinsfile_dir + &quot;/Jenkinsfile&quot;
+
+          //Read the Jenkins File
+          def curl_cmd = &apos;&apos;
```
Comment (suryajak) [inline installscripts/jazz-terraform-unix-noinstances/scripts/gitlab-trigger-job.sh]: There were multiple bugs with this code that we had fixed as part of a PR in core for gitlabs. Now the same bugs are showing up here, so please review it and let me know if this required here.

File: installscripts/cookbooks/jenkins/files/node/jenkins-conf.properties (hunk 1 above)
Comment (Feroz-Shaikh) [reply installscripts/cookbooks/jenkins/files/node/jenkins-conf.properties]: Removed*

File: installscripts/jazz-terraform-unix-noinstances/scripts/gitlab-trigger-job.sh (hunk 2 above)
Comment (Feroz-Shaikh) [reply installscripts/jazz-terraform-unix-noinstances/scripts/gitlab-trigger-job.sh]: Updated the code to handle this as a Jenkinsfile. 
But this file was merged to branch v1.4 in core. Hence, I couldn't find it in current active 1.3.1. All the gitlab core changes are a part of v1.4 in core. Are these changes added to v1.3.1 in jazz-core as well? Can you please address this.

File: installscripts/jazz-terraform-unix-noinstances/scripts/gitlab-trigger-job.sh (hunk 3 above)
Comment (Feroz-Shaikh) [reply installscripts/jazz-terraform-unix-noinstances/scripts/gitlab-trigger-job.sh]: addressed.

**Comment by Feroz-Shaikh** [comment]
@suryajak  Addressed the review comments in PR https://github.com/tmobile/jazz-installer/pull/89

**Pull request merged**